packages/client/test/rpc/engine/newPayloadV1.spec.ts

```diff
@@ -99,6 +99,8 @@ tape(`${method}: call with non existent parent hash`, async (t) => {
 tape(
   `${method}: call with unknown parent hash to store in remoteBlocks, then call valid ancestor in fcU`,
   async (t) => {
+    // @ts-ignore -- removes sharding hardfork since this test expects the hardfork to be merge
+    delete genesisJSON.config['shardingForkTime']
```

Comment:
i think we can remove this line as i separated the jsons of sharding from post-merge

Reply:
Will do.